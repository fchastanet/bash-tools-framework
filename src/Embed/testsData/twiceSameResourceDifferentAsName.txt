# INCLUDE '${dir}/testsData/binaryFile' as 'valid1'
# INCLUDE '${dir}/testsData/binaryFile' as 'valid2'
