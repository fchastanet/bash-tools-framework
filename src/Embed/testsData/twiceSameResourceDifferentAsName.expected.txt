Embed::extractFile_valid1() {
  if [[ ! -f "${PERSISTENT_TMPDIR:-/tmp}/bin/valid1" ]]; then
    mkdir -p "$(dirname "${PERSISTENT_TMPDIR:-/tmp}/bin/valid1")"
    base64 -d >"${PERSISTENT_TMPDIR:-/tmp}/bin/valid1" <<<"IyEvdXNyL2Jpbi9lbnYgYmFzaAoKZWNobyAiYmluYXJ5RmlsZSAkKiIK"
    chmod "755" "${PERSISTENT_TMPDIR:-/tmp}/bin/valid1"
  fi
}

Embed::extractFile_valid1

declare -gx embed_file_valid1="${PERSISTENT_TMPDIR:-/tmp}/bin/valid1"
Embed::extractFile_valid2() {
  if [[ ! -f "${PERSISTENT_TMPDIR:-/tmp}/bin/valid2" ]]; then
    mkdir -p "$(dirname "${PERSISTENT_TMPDIR:-/tmp}/bin/valid2")"
    base64 -d >"${PERSISTENT_TMPDIR:-/tmp}/bin/valid2" <<<"IyEvdXNyL2Jpbi9lbnYgYmFzaAoKZWNobyAiYmluYXJ5RmlsZSAkKiIK"
    chmod "755" "${PERSISTENT_TMPDIR:-/tmp}/bin/valid2"
  fi
}

Embed::extractFile_valid2

declare -gx embed_file_valid2="${PERSISTENT_TMPDIR:-/tmp}/bin/valid2"
